_includes/navbar.html

```diff
@@ -56,42 +53,22 @@
                         </ul>
                     </li>
                     <li class="dropdown" role="none">
-                        <a role="menuitem"
-                           aria-haspopup="true"
-                           aria-expanded="false"
-                           class="dropdown-toggle"
-                           data-toggle="dropdown"
-                           data-target="#"
-                           href="#">
-                            Tools <b class="caret"></b>
-                        </a>
-                        <ul class="dropdown-menu" role="menu" aria-label="Tools dropdown.">
-                            <li role="none">
-                                <a role="menuitem"
-                                   href="https://marketplace.visualstudio.com/items?itemName=stansw.vscode-odata"
-                                   target="_blank">
-                                    OData for Visual Studio Code
-                                </a>
-                            </li>
-                            <li role="none">
-                                <a role="menuitem" href="https://pragmatiqa.com/xodata/" target="_blank">XOData</a>
-                            </li>
-                        </ul>
+                        <a role="menuitem" href="/tools">Tools</a>
                     </li>
                     <li role="none">
-                        <a role="menuitem"  href="/ecosystem">Ecosystem</a>
+                        <a role="menuitem" href="/blog">Blog</a>
                     </li>
                     <li role="none">
-                        <a role="menuitem" href="/contribution">Getting Involved</a>
+                        <a role="menuitem" href="/ecosystem">Ecosystem</a>
                     </li>
                 </ul>
 
-                <form class="navbar-form navbar-left" role="search">
-                    <div class="form-group combobox combobox-list">
+                <form class="navbar-form navbar-right search-bar" role="search" style="margin-left: auto">
+                    <div class="search-bar form-group combobox combobox-list">
                         <input id="main-search-input"
                                type="text"
                                aria-label="Search"
-                               class="form-control st-default-search-input"
+                               class="search-bar-input form-control"
```

Comment:
Check to see if the swift type works I doubt it would fail with the css change but probably requires some verification would be required as the js provides accessible search through the dynamic widget.

Reply:
As far as I can tell, the class change doesn't affect the functionality (the scripts appear reference the `main-search-input` ID, and the search results seem the same), but if there's something I'm missing, do let me know.